### DISCUSSION TITLE
Folder icon/ folder cover icon

### DISCUSSION THREAD

**Discussion by MohamedxSalah** [LawnchairLauncher/lawnchair#3021] [Ideas]
I have been waiting for folder icon option for lawnchair since lawnchair 11

In the past there used to be an option to choose custom icon for the folder
Or to make the folder take (mask)  the shape (icon) of the first app inside the folder (folder cover/mask icon) 

Which used to make the home screen setup a ton more sleeker and better

Please make this feature available again

**Comment by StarGW-net** [comment]
I agree, its a key usability feature that is now missing.

**Comment by iFabse** [comment]
I use the old version of lawnchair because of that. If this is going to be implemented in V14 I would switch

**Comment by slhale** [comment]
I had to redo my home screen setup to not use folders at all because the folders appear way too big compared to the play store lawnchair. I would appreciate this feature back

**Comment by brainchild0** [comment]
The "cover" option for desktop folders is an essential feature, for anyone juggling an application ensemble more elaborate than the core basics, yet it has been dropped from earlier releases. Its absence is the reason I have avoided upgrading from earlier versions of the application. 

However, Issue #4745 represents another essential feature, which will be available only for the newer releases, for those wishing to use the corresponding feature from the system, called *Private Spaces*.

Currently, the lack of availability for both features in a single release represents a seriously inconvenient dilemma.

**Comment by Steffen777** [comment]
I would like to suggest an additional folder icon layout similar to Nova Launcher.

Instead of showing several small app icons in a grid, the folder could display a few larger, overlapping app icons. One app could be shown more prominently to visually represent the folder's content, while other apps remain partially visible behind it.

<img width="220" height="381" alt="Screenshot_20260829-090859~2" src="https://github.com/user-attachments/assets/6401947f-e8b2-4538-baad-3f7485c8f423" />


I find this layout much cleaner and calmer, especially when using many folders. It makes the folder content easier to recognize at a glance without displaying many tiny icons.

It would be great to have this as an additional folder preview option in Lawnchair.

Thank you for  it!

**Comment by brainchild0** [comment]
I like this approach. It expresses one icon as most prominent, without obscuring that multiple of icons are associated with the folder. However, suggesting the alternative aesthetic sidesteps the question of whether the folder is opened by a tap versus a swipe, in the latter case with a tap immediately launching the first member of the folder.
